src/SupplierManager/Travely.SupplierManager.API/Startup.cs

```diff
@@ -0,0 +1,107 @@
+using System;
+using AutoMapper;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.OpenApi.Models;
+using TourEntities.Service.Accommodation;
+using Travely.Common.Extensions;
+using Travely.Common.Grpc;
+using Travely.Common.Grpc.Abstraction;
+using Travely.Common.ServiceDiscovery;
+using Travely.Common.Swagger;
+using Travely.Shared.IdentityClient.Authorization.Config;
+using Travely.SupplierManager.Grpc;
+using Travely.SupplierManager.Grpc.Client.Abstraction;
+using Travely.SupplierManager.Grpc.Client.Implementation;
+using Travely.SupplierManager.Grpc.Settings;
+using Travely.SupplierManager.Repository.DbContexts;
+using Travely.SupplierManager.Repository.Entities;
+using Travely.SupplierManager.Repository;
+using Travely.SupplierManager.Service;
+using Travely.SupplierManager.Service.Helpers;
+
+namespace Travely.SupplierManager.API
+{
+    public class Startup
+    {
+        public Startup(IConfiguration configuration, IWebHostEnvironment environment)
+        {
+            Configuration = configuration;
+            Environment = environment;
+        }
+
+        public IConfiguration Configuration { get; }
+
+        public IWebHostEnvironment Environment { get; }
+
+        // This method gets called by the runtime. Use this method to add services to the container.
+        public void ConfigureServices(IServiceCollection services)
+        {
+            services.AddControllers();
+            
+            services.AddGrpc();
+
+            services.AddDbContext<SupplierDbContext>(options =>
+                options.UseSqlServer(Configuration.GetConnectionString("SupplierDbContext")));
+            
+            // TODO move from here
+            services.AddScoped<ISupplierRepository<AccommodationEntity>, SupplierRepository<AccommodationEntity>>();
+            services.AddScoped<ISupplierService<Accommodation>, SupplierService<Accommodation, AccommodationEntity>>();
+            services.AddScoped<ISupplierManagerClient, SupplierManagerClient>();
+            services.AddSingleton<ISearchHelper<AccommodationEntity>, AccommodationSearchHelper>();
+            
+            services.AddSingleton<ISortHelper<AccommodationEntity>, SortHelper<AccommodationEntity>>();
+            
+            services.Configure<GrpcSettings<SupplierProto.SupplierProtoClient>>(
+                Configuration.GetSection("SupplierGrpcService"));
+            services
+                .AddScoped<IServiceSettingsProvider<SupplierProto.SupplierProtoClient>,
+                    SupplierManagerSettingsProvider>();
```

Comment:
Move to `SupplierServiceCillectionExtensions` extensions class  in the same layer where contains those services